lib/ansible/executor/task_executor.py

```diff
@@ -605,6 +605,7 @@ def _get_connection(self, variables, templar):
                 try:
                     cmd = subprocess.Popen(['ssh','-o','ControlPersist'], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
                     (out, err) = cmd.communicate()
+                    err = to_str(err)
```

Comment:
That can throw tracebacks without a lot of control possibilities from us.  Using to_unicode and to_bytes is better.  Since we're dealing with it as text in this case, to_unicode is what we should be using.